templates/pimcore5/files/.platform/routes.yaml

```diff
@@ -0,0 +1,14 @@
+# The routes of the project.
+#
+# Each route describes how an incoming URL is going
+# to be processed by Platform.sh.
+
+"https://{default}/":
+    type: upstream
+    upstream: "pimcore:http"
+    cache:
+      enabled: false
```

Comment:
Don't disable the cache.  If anything it should be configured properly to whitelist their session cookie(s).